src/components/organisms/PostInputForm/PostInputForm.style.ts

```diff
@@ -25,21 +24,12 @@ export const bodyStyle = css({
 
 export const optionStyle = css({
   display: 'flex',
-  alignItems: 'center',
   columnGap: '17px',
 });
 
-export const inputContainerStyle = css(typo.Main.Medium, {
+export const inputStyle = css(typo.Main.Medium, {
```

Comment:
```suggestion
export const inputStyling = css(typo.Main.Medium, {
```
과 같이 기존 스타일링 네이밍과 싱크를 맞추는 것도 좋아보여요! 어느까지나 취향입니다!